backend/app/notify/webhook_test.go

```diff
@@ -0,0 +1,128 @@
+package notify
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/umputun/remark42/backend/app/store"
+)
+
+func TestWebhook_New(t *testing.T) {
+
+	ms := newMockWebhookServer()
+	defer ms.Close()
+
+	wh, err := ms.newClient(WebhookParams{
+		WebhookURL: ms.URL + "/webhook",
+		Headers:    "Authorization:Basic AXVubzpwQDU1dzByYM==",
+		Timeout:    1 * time.Second,
+	})
+	assert.NoError(t, err)
+	assert.NotNil(t, wh)
+
+	assert.Equal(t, ms.URL+"/webhook", wh.WebhookURL)
+	assert.Equal(t, "Authorization:Basic AXVubzpwQDU1dzByYM==", wh.Headers)
+	assert.Equal(t, 1*time.Second, wh.Timeout)
+	assert.Equal(t, `{"text": "{{.Text}}"}`, wh.Template)
```

Comment:
After you'll move the template definition to NewWebhook, please also add a test case on error on template which doesn't compile.

Reply:
Tested